pkg/mflag/flag.go

```diff
@@ -105,6 +105,44 @@ var ErrRetry = errors.New("flag: retry")
 // -- bool Value
 type boolValue bool
 
+var (
+	m_Daemon_Flags  map[string]bool
+	Has_Daemon_Flag bool
+)
+
+func init() {
+	m_Daemon_Flags = make(map[string]bool)
```

Comment:
I wish there was another way to do this because this is begging for this list to be out of sync.

Could you put this list in alphabetical order so its easier to visually compare it with the help text to see if something is missing?